pkg/controller/controllercmd/builder.go

```diff
@@ -70,6 +70,10 @@ type ControllerBuilder struct {
 	authenticationConfig *operatorv1alpha1.DelegatedAuthentication
 	authorizationConfig  *operatorv1alpha1.DelegatedAuthorization
 	healthChecks         []healthz.HealthChecker
+
+	// nonZeroExitFn exists for unit testing and should be not used outside unit tests
+	// The signature of this function matches klog.Fatalf.
+	nonZeroExitFn func(args ...interface{})
```

Comment:
klog.Fatalf has a string as first arg, no?

Reply:
no :-)

Reply:
"Fatal" does not.

Reply:
you comment is wrong

Reply:
fixed